django/contrib/admindocs/views.py

```diff
@@ -145,13 +156,29 @@ def get_context_data(self, **kwargs):
 class ViewDetailView(BaseAdminDocsView):
     template_name = 'admin_doc/view_detail.html'
 
-    def get_context_data(self, **kwargs):
-        view = self.kwargs['view']
+    @staticmethod
+    def _get_view_func(view):
         urlconf = get_urlconf()
         if get_resolver(urlconf)._is_callback(view):
             mod, func = get_mod_func(view)
-            view_func = getattr(import_module(mod), func)
-        else:
+            try:
+                return getattr(import_module(mod), func)
+            except ImportError:
+                # Unable to import the module - try processing it again to get the class.
```

Comment:
I'm having trouble understanding which case this is from the comment (which by the way, we wrap at 79 characters). Maybe you could add an example of what mod/func look like for each case.